app/views/layouts/application.html.erb

```diff
@@ -6,8 +6,12 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
     <%= csrf_meta_tags %>
   </head>
-
   <body>
+    <nav>
+      <ul>
+        <li><%= link_to "Sign Out", logout_path, method: :delete %></li>
+      </ul>
+    </nav>
```

Comment:
This shouldn't be a static link in this file. Are you only doing that for the sake of signing out? I think we should implement it so that it switches to "Sign In" if there is no `current_user`.